src/environmentd/src/http/metrics.rs

```diff
@@ -44,19 +44,19 @@ impl Metrics {
                 name: "requests_total",
                 help: "Total number of http requests received since process start.",
                 subsystem: component,
```

Comment:
What do you think about using "subsystem" instead of adding an separate tag? We currently use subsystem to disambiguate between the "internal" and "external" http servers

Reply:
That would make these separate metrics. I like the property that they are the same metric, but with different labels? It makes them groupable in promql.

Reply:
Makes sense!